lib/actions.ex

```diff
@@ -228,7 +228,12 @@ defmodule ExTus.Actions do
   def post(conn, create_cb) do
     headers = Utils.read_headers(conn)
 
-    upload_type = headers["upload-type"]
+    # Identify path for Profile Image
+    upload_type = if conn.request_path == "/profile-image" do
```

Comment:
change here as well....as we will use `/profiles` as path

Reply:
done